electrophysiology/models.py

```diff
@@ -106,21 +118,26 @@ class SpikeSortedUnit(BaseExperimentalData):
     channel_group = models.IntegerField(null=True, blank=True,
                                         help_text="which shank this unit came from (an integer not a link)")
     trough_to_peak_width = models.FloatField(null=True, blank=True,
-                                 help_text="ms, computed from unfiltered mean waveform. TODO: which waveform?")
+                                 help_text="ms, computed from unfiltered mean spike waveform.")
     half_width = models.FloatField(null=True, blank=True,
-                                 help_text="ms, half width of negative peak in unfiltered waveform. TODO: which waveform?")
+                                 help_text="ms, half width of negative peak in unfiltered spike waveform.")
     trough_to_peak_amplitude = models.FloatField(null=True, blank=True,
-                                 help_text="µV, from filtered waveform. TODO: which waveform?")
+                                 help_text="µV, from filtered spike waveform.")
+								 # when you say which waveform, do you mean which channel? I guess the peak, but we can leave it unspecified...
```

Comment:
Yep. Should it be peak, or interpolated inter-electrode peak (assuming the cell is between two sites?) I'd have thought some measure like calculated spike CSD might be more useful since this is amplitude is surely mostly dependent on how close the cell is to the probe rather than revealing a whole lot of information about the cell?